lua/lspconfig.lua

```diff
@@ -7,41 +7,41 @@ local M = {
 ---@class Alias
 ---@field to string The new name of the server
 ---@field version string The version that the alias will be removed in
----@param name string
----@return Alias
-local function server_alias(name)
-  local aliases = {
-    ['fennel-ls'] = {
-      to = 'fennel_ls',
-      version = '0.2.0',
-    },
-    ruby_ls = {
-      to = 'ruby_lsp',
-      version = '0.2.0',
-    },
-    ['starlark-rust'] = {
-      to = 'starlark_rust',
-      version = '0.2.0',
-    },
-    sumneko_lua = {
-      to = 'lua_ls',
-      version = '0.2.0',
-    },
-    tsserver = {
-      to = 'ts_ls',
-      version = '0.2.0',
-    },
-  }
+---@field inconfig? boolean need shown in lspinfo
+local aliases = {
+  ['fennel-ls'] = {
+    to = 'fennel_ls',
+    version = '0.2.1',
+  },
+  ruby_ls = {
+    to = 'ruby_lsp',
+    version = '0.2.1',
+  },
+  ['starlark-rust'] = {
+    to = 'starlark_rust',
+    version = '0.2.1',
+  },
+  sumneko_lua = {
+    to = 'lua_ls',
+    version = '0.2.1',
+  },
+  tsserver = {
+    to = 'ts_ls',
+    version = '0.2.1',
+  },
+}
 
-  return aliases[name]
+M.server_aliases = function()
+  return aliases
 end
 
 local mt = {}
 function mt:__index(k)
   if configs[k] == nil then
-    local alias = server_alias(k)
+    local alias = M.server_aliases()[k]
```

Comment:
if we return the entire list we didn't encapsulate anything, and the internal list can be modified.

```
---@return Alias
local function server_alias(name)
```

Reply:
then in lspinfo there need a special param like `inconfig`  .. because there need a hole entry

Reply:
oh can be just nil